pyfakefs/tests/all_tests_without_extra_packages.py

```diff
@@ -0,0 +1,19 @@
+import sys
```

Comment:
I'm not very happy with the approach in this file but don't have a better idea.

Reply:
Unfortunately `pytest` depends on `pathlib2` which depends on `scandir`!

Reply:
Aha I see what's happening, their [setup.py](https://github.com/pytest-dev/pytest/blob/master/setup.py) adds `pathlib2` as a dependency when Python < 3.6 so if we find no other approach I'll update the comment in `extra_requirements.txt`.

Reply:
Ah, I see - well, I can't think of another approach right now in this case. Maybe @jmcgeheeiv has an idea, otherwise I would it leave as is.